tools/ds-identify

```diff
@@ -989,8 +989,12 @@ dscheck_NoCloud() {
     fi
 
     # This is a bit hacky, but a NoCloud false positive isn't the end of the world
-    if check_config "NoCloud" && check_config "user-data" && check_config "meta-data"; then
-        return ${DS_FOUND}
+    if check_config "NoCloud"; then
+        if check_config "user-data" && check_config "meta-data"; then
+            return ${DS_FOUND}
+        elif check_config "seedfrom"; then
```

Comment:
`DataSourceOVF` does also use `seedfrom`, as `check_config` does effectively a grep-search, should check for the abscence of OVF in sys-config files?

```suggestion
        elif ! check_config "OVF" && check_config "seedfrom"; then
```

Reply:
Good point, that seems reasonable.

Reply:
@aciba90 actually, no I don't think that doing this makes sense. The seedfrom used by DataSourceOVF comes through the xml file, not system configuration.